### DISCUSSION TITLE
Number of topics (k) in LDA method

### DISCUSSION THREAD

**Discussion by NAsic123** [biolab/orange3-text#879] [Ideas]
In Orange, you have to type the number of topics you want to have in the LDA model. But what if you do not know the exact/optimal number of topics? How do we calculate the optimal number of topics (k) in the LDA method?
I found these useful metrics and want to know if anyone from you knows how to calculate these metrics in Orange?

https://cran.r-project.org/web/packages/ldatuning/vignettes/topics.html

**Comment by ajdapretnar** [comment]
There's not such option. See https://github.com/biolab/orange3-text/issues/880.
